### PR TITLE
JAPI-80 Changes for dropzones with new hostname/address combinations

### PR DESCRIPTION
**Pull request by Michael-Gardner**
Signed-off-by: Michael Gardner <michael.gardner@lexisnexisrisk.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).

## Checklist:
- [x] I have created a corresponding JIRA ticket for this submission
- [x] My code follows the code style of this project.
  - [ ] I have applied the Eclipse code-format template provided.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the HPCC Systems CONTRIBUTORS document (https://github.com/hpcc-systems/HPCC-Platform/wiki/Guide-for-contributors).
- [x] The change has been fully tested:
  - [ ] I have performed unit tests to cover my changes.
  - [ ] I have performed system test and covered possible regressions and side effects.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.

### CODE REVIEW

**Comment by Michael-Gardner** [comment]
@rpastrana adds a getConfNetAddresses and getNetAddresses functions

File: wsclient/src/main/java/org/hpccsystems/ws/client/platform/DropZone.java [hunk 1]
```diff
@@ -49,6 +50,35 @@ public String getName()
...
+    		for(int i = 0; i < info.getTpMachines().length; i++)
+    		{
+    			naList[i] = new String(info.getTpMachines()[i].getNetaddress());
+    		}
+    		return naList;
+    	}
+    	return null;
+    }
+    
+    // Deprecated
```
Comment (rpastrana) [inline wsclient/src/main/java/org/hpccsystems/ws/client/platform/DropZone.java]: Use the Java notation:
/**
 * 
 * @deprecated Do not use this method!
 */
 @Deprecated

File: wsclient/src/main/java/org/hpccsystems/ws/client/platform/DropZone.java [hunk 2]
```diff
@@ -49,6 +50,35 @@ public String getName()
         return info.getName();
     }
 
+    public String[] getConfNetAddresses()
```
Comment (rpastrana) [inline wsclient/src/main/java/org/hpccsystems/ws/client/platform/DropZone.java]: let's go ahead and expand the name so there's no doubt about what it does (getConfiguredNetAddresses), also let's add a method comment.
In the comment we can distinguish this method from getNetAddresses

File: wsclient/src/main/java/org/hpccsystems/ws/client/platform/DropZone.java [hunk 3]
```diff
@@ -49,6 +50,35 @@ public String getName()
...
+    		for(int i = 0; i < info.getTpMachines().length; i++)
+    		{
+    			cnaList[i] = new String(info.getTpMachines()[i].getConfigNetaddress());
+    		}
+    		return cnaList;
+    	}
+    	return null;
+    }
+    
+    public String[] getNetAddresses()
```
Comment (rpastrana) [inline wsclient/src/main/java/org/hpccsystems/ws/client/platform/DropZone.java]: let's add a meth comment

File: wsclient/src/test/java/org/hpccsystems/ws/client/platform/test/PlatformTester.java [hunk 4]
```diff
@@ -203,7 +204,31 @@ else if (currentParam.matches(WSSQLPORTPATTERN))
...
+            			for(int b = 0; b < cna.length; b++)
+            			{
+            				System.out.println("\tconfNetAddress: " + cna[b]);
+            				System.out.println("\tnetAddress:     " + na[b]);
+            				System.out.println("\t-------------------------------------");
+            			}
+            		}
+            	}
+            }
+
```
Comment (rpastrana) [inline wsclient/src/test/java/org/hpccsystems/ws/client/platform/test/PlatformTester.java]: can we also add a test using HPCCFileSprayClient.listFiles? HPCCFileSprayClient.fetchLocalDropZones, HPCCFileSprayClient.fetchDropZones(string), etc to make sure that functionality continues to work w/ the new DZ structures?

File: wsclient/src/main/java/org/hpccsystems/ws/client/platform/DropZone.java (hunk 2 above)
Comment (Michael-Gardner) [inline wsclient/src/main/java/org/hpccsystems/ws/client/platform/DropZone.java]: Great idea.

**Pull request merged**